cli/src/test_highlight.rs

```diff
@@ -162,68 +162,7 @@ pub fn test_highlight(
     let assertions =
         parse_position_comments(highlighter.parser(), highlight_config.language, source)?;
 
-    iterate_assertions(&assertions, &highlights, &highlight_names)?;
```

Comment:
I deleted all this since the code is the same as `iterate_assertions`, seemed to be an accidental duplicate that got refactored into a function